examples/guestbook-go/v1beta3/guestbook-service.json

```diff
@@ -0,0 +1,19 @@
+{
+    "kind": "Service",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "guestbook",
+        "labels": {
+            "name": "guestbook"
+        }
+    },
+    "spec": {
+        "port": 3000,
+        "containerPort": "http-server",
+        "protocol": "TCP",
+        "createExternalLoadBalancer": true,
```

Comment:
We're not creating an external load balancer in the existing examples because it's not supported by all cloud providers.